ensime_shared/debugger.py

```diff
@@ -54,6 +54,31 @@ def debug_continue(self, args, range=None):
             "typehint": "DebugContinueReq",
             "threadId": self.debug_thread_id})
 
+    def debug_step(self, args, range=None):
+        self.log("debug_step: in") 
+        self.send_request({
+            "typehint":"DebugStepReq",
+            "threadId":self.debug_thread_id})
+ 
+    def debug_step_out(self, args, range=None):
+        self.log("debug_step_out: in") 
+        self.send_request({
+            "typehint":"DebugStepOutReq",
+            "threadId":self.debug_thread_id})
+ 
+    def debug_next(self, args, range=None):
+        self.log("debug_next: in") 
+        self.send_request({
+            "typehint":"DebugNextReq",
+            "threadId":self.debug_thread_id})
+
+    def debug_locate_name(self, args, range=None):
```

Comment:
This returns a `DebugLocation`, which is unhandled right now. Please implement a handler for this function or add `debug_locate_name` in a  separate PR.